source/Assets/Scripts/InfinarioSDK/Infinario.cs

```diff
@@ -15,6 +15,11 @@ private Infinario()
 		{
 			//Prepare for wrappers
 			implementation = new SDK.Unity ();
+
+			//Setup decimal separator for numbers
+            var customCulture = (System.Globalization.CultureInfo)System.Threading.Thread.CurrentThread.CurrentCulture.Clone();
```

Comment:
Indentation here (spaces) does not match the surrounding code (tabs). Please use tabs so the formatting is consistent.